jackson-dataformat-cbor : class not found exception

when not using any parser for format converter, the following exception happens but that exception is ignored, printing "Retriable send error".
to further diagnose the problem, I had to add stack trace in OnSendError().
```
com.amazon.kinesis.streaming.agent.tailing.AsyncPublisher [ERROR] stack trace : 
com.amazonaws.AmazonClientException: Unable to marshall request to JSON: com/fasterxml/jackson/dataformat/cbor/CBORFactory
	at com.amazonaws.services.kinesis.model.transform.PutRecordsRequestMarshaller.marshall(PutRecordsRequestMarshaller.java:97)
	at com.amazonaws.services.kinesis.AmazonKinesisClient.putRecords(AmazonKinesisClient.java:1733)
	at com.amazon.kinesis.streaming.agent.tailing.KinesisSender.attemptSend(Unknown Source)
	at com.amazon.kinesis.streaming.agent.tailing.AbstractSender.sendBuffer(Unknown Source)
	at com.amazon.kinesis.streaming.agent.tailing.SimplePublisher.sendBufferSync(Unknown Source)
	at com.amazon.kinesis.streaming.agent.tailing.AsyncPublisher.access$001(Unknown Source)
	at com.amazon.kinesis.streaming.agent.tailing.AsyncPublisher$1.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NoClassDefFoundError: com/fasterxml/jackson/dataformat/cbor/CBORFactory
	at com.amazonaws.protocol.json.SdkJsonProtocolFactory.getSdkFactory(SdkJsonProtocolFactory.java:99)
	at com.amazonaws.protocol.json.SdkJsonProtocolFactory.createGenerator(SdkJsonProtocolFactory.java:54)
	at com.amazonaws.services.kinesis.model.transform.PutRecordsRequestMarshaller.marshall(PutRecordsRequestMarshaller.java:65)
	... 9 more
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.dataformat.cbor.CBORFactory
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 12 more

```

As we can see in the previous stack trace, the problem was ClassNotFoundException, missing the following artifact

- com/fasterxml/jackson/dataformat/jackson-dataformat-cbor

setup script should download that artifact to resolve the problem.

--- a/setup
+++ b/setup
@@ -95,6 +95,7 @@ download_dependencies() {
                   com.fasterxml.jackson.core:jackson-annotations:2.6.3 \
                   com.fasterxml.jackson.core:jackson-core:2.6.3 \
                   com.fasterxml.jackson.core:jackson-databind:2.6.3 \
+                  com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.6.3 \
                   com.google.code.findbugs:jsr305:3.0.1 \
                   com.google.guava:guava:18.0 \
                   org.apache.httpcomponents:httpclient:4.5.1 \

